.github/generate_dev_tests.py

```diff
@@ -34,12 +34,23 @@
         yaml.dump(
             workflox_yml,
             outfile,
-            default_style="|",
             default_flow_style=False,
             allow_unicode=True,
             width=float("inf"),
             sort_keys=False,
         )
+    
+    with open(Path("workflows", new_name), "r+") as outfile:
+        workflox_yml = outfile.read()
+        workflox_yml = workflox_yml.replace("'", "")
+        workflox_yml = workflox_yml.replace("run:", "run: |\n       ")
+
+        workflox_yml = "\n".join([ll.rstrip() for ll in workflox_yml.splitlines() if ll.strip()])
+
+        outfile.seek(0)
+        outfile.write(workflox_yml)
+        outfile.truncate()
+
 
     with open(Path("workflows", new_name), "r+") as outfile:
```

Comment:
This code block could be merged with the one right above

Reply:
Thanks, fixed!